client/state/themes/utils.js

```diff
@@ -216,7 +216,7 @@ export function isThemeMatchingQuery( query, theme ) {
 	const queryWithDefaults = { ...DEFAULT_THEME_QUERY, ...query };
 	return every( queryWithDefaults, ( value, key ) => {
 		switch ( key ) {
-			case 'search':
+			case 'search': {
```

Comment:
Nitpick: while I appreciate the usage of braces in `switch` statements, I haven't seen this pattern used anywhere else in the Calypso codebase and would drop it just for the sake of consistency.

Reply:
This one was causing a linter error, which is why I added the braces. I think new linter rules must be enforcing braces around blocks which are complex enough -- I think with that, it's ok to keep it?

Reply:
TIL! Yeah, let's keep it in that case.